server/WebAPI/DTO/GenreDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.DTO
+{
+    public class GenreDTO
+    {
+        public int GenreId { get; set; }
+        public string GenreName { get; set; }
```

Comment:
it is GenreDTO, we don't need specify that it is id and name of Genre - just Id and Name